### PR TITLE
Eval Villain Update 2022-02-15

### PR DESCRIPTION
**Pull request by zapbot**
Updates based on https://addons.mozilla.org/firefox/addon/eval-villain/

Signed-off-by: zapbot <12745184+zapbot@users.noreply.github.com>

### CODE REVIEW

**Review by psiinon — APPROVED** [review]

**Comment by kingthorin** [comment]
Set DCO pass as I needed to tweak via github.dev

File: addOns/evalvillain/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Changed
```
Comment (thc202) [inline addOns/evalvillain/CHANGELOG.md:7]: `Fixed`

File: addOns/evalvillain/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,9 @@ All notable changes to this add-on will be documented in this file.
...
+- Fix the downloaded version of the Eval Villain Firefox extension (Issue 7008).
```
Comment (thc202) [inline addOns/evalvillain/CHANGELOG.md:8]: The referenced issue is about Chrome not this specific issue, might worth tweaking.

**Comment by thc202** [comment]
The version could be changed to `0.1.1`.

**Comment by kingthorin** [comment]
Okay will tweak via Eclipse shortly. Just waiting for a meeting to wrap.

**Comment by kingthorin** [comment]
Tweaked.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**